internal/hcs/system.go

```diff
@@ -326,13 +336,113 @@ func (computeSystem *System) Properties(ctx context.Context, types ...schema1.Pr
 	return properties, nil
 }
 
+// statisticsInProc emulates what HCS does to grab statistics for a given container with a small
+// change to make grabbing the private working set total much more efficient.
+func (computeSystem *System) statisticsInProc(ctx context.Context) (*hcsschema.Properties, error) {
+	// See if we'd even be able to open the silo. We'll need to be local
+	// system/system so check first.
+	usr, err := windows.GetCurrentProcessToken().GetTokenUser()
+	if err != nil {
+		return nil, err
+	}
+	if !usr.User.Sid.IsWellKnown(windows.WinLocalSystemSid) {
```

Comment:
Instead of checking this, can we just try to open and return an error if it fails?

Reply:
Yea, I guess it's kind of redundant. And any error is probably cause for aborting, I was just going to check for access is denied originally